Emby.Server.Implementations/Data/BaseSqliteRepository.cs

```diff
@@ -12,172 +11,76 @@ namespace Emby.Server.Implementations.Data
     public abstract class BaseSqliteRepository : IDisposable
     {
         protected string DbFilePath { get; set; }
-        protected ReaderWriterLockSlim WriteLock;
 
-        protected ILogger Logger { get; private set; }
+        protected ILogger Logger { get; }
 
         protected BaseSqliteRepository(ILogger logger)
         {
             Logger = logger;
-
-            WriteLock = new ReaderWriterLockSlim(LockRecursionPolicy.NoRecursion);
         }
 
         protected TransactionMode TransactionMode => TransactionMode.Deferred;
 
         protected TransactionMode ReadTransactionMode => TransactionMode.Deferred;
 
-        internal static int ThreadSafeMode { get; set; }
-
-        static BaseSqliteRepository()
-        {
-            SQLite3.EnableSharedCache = false;
-
-            int rc = raw.sqlite3_config(raw.SQLITE_CONFIG_MEMSTATUS, 0);
-            //CheckOk(rc);
+        protected virtual ConnectionFlags DefaultConnectionFlags => ConnectionFlags.SharedCached | ConnectionFlags.NoMutex;
```

Comment:
SharedCache necessary now we've only one connection per db?

Reply:
Doesn't hurt anything right?

Reply:
hurt, no, performance, maybe.. but think you've removed it now anyway?